ui/src-tauri/src/main.rs

```diff
@@ -200,6 +203,19 @@ impl PasswordStore {
     }
 }
 
+/// Sends the current `password` into storage from the UI.
+#[tauri::command]
+async fn send_recovery_info(
+    reset_handle: State<'_, ResetHandle>,
+    phrase: String,
+    password: String,
+) -> Result<bool, ()> {
+    let info = (Secret::new(phrase), Secret::new(password));
+
+    reset_handle.send(info).await.expect("Reset channel broke");
+    Ok(true)
```

Comment:
Add a `from_strings` (bikeshed the name) method to the `ResetInfo` struct that takes `Strings` and converts them into `Secret`s. Also, return the error back up to the caller.

```suggestion
    reset_handle.send(ResetInfo::from_strings(phrase, password)).await
```